Libraries/Pressability/Pressability.js

```diff
@@ -445,6 +452,21 @@ export default class Pressability {
         }
       },
     };
+    
+    const keyEventHandlers = {
+      onKeyDown: (event: KeyEvent): void => {
```

Comment:
For extra context, react-native-windows did this a slightly different way. There, we translate key events seen by Pressability into press events, instead of having Pressability emit key events. This matches some legacy behavior, but it also means press event shape is unexpected when press happens by keyboard activation.

If you want raw pass through, it might be simpler to just avoid passing the handlers from Pressable/Touchable to Pressability.

Reply:
Do we get state change visuals on keyboard with this? IIRC hooking into the Pressability state machine gave us those for free.

Reply:
This mimics the pattern we used for onFocus and onBlur for macOS. What do you mean by state change visuals on keyboard?

Reply:
If I understand properly, windows treats onKeyUp as onPressIn, and onKeyDown as onPressOut. Then, any visuals that might be associated with the pressed state of a perusable also happen on key press. This sounds like a desirable behavior that I think we'd want to incorporate...